### PR TITLE
Api-connect

### PR DESCRIPTION
**Pull request by Naledi-Dikgale**
**In this Pull Request I did the following**

- Make use of the Bookstore API as my backend.
- Use Axios to make your HTTP requests to the backend
- Fetch books from the Bookstore API after the component has rendered
- Refactor the add book and remove book features to persist changes in the server
- Application should behave the same way after the refactoring

### CODE REVIEW

File: src/components/BookForm.jsx [hunk 1]
```diff
@@ -21,7 +21,15 @@ const BookForm = () => {
     event.preventDefault();
     const { title, author } = formState;
     if (title.trim() !== '' && author.trim() !== '') {
-      dispatch(addBook(formState));
+      const formInput = {
+        // eslint-disable no-use-before-define
```
Comment (Shisui6) [inline src/components/BookForm.jsx]: Please try to solve linter errors without the use of eslint-disable. The linters are in place to enforce best practices in coding and learning to solve them will make you a better developer👍

File: src/redux/Books/bookSlice.js [hunk 2]
```diff
@@ -1,27 +1,35 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const BaseUrl = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/uMfxud1McryQAfRNygOp/books';
+// const endpoint = "/apps/uMfxud1McryQAfRNygOp/books'";
+// const BaseUrl = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/UHXf29hQzudmQrt1PhXv/books';
```
Comment (Shisui6) [inline src/redux/Books/bookSlice.js]: Please remove all commented out code from your codebase to keep it clean👍

**Review by Shisui6 — CHANGES_REQUESTED** [review]
Hi @Naledi-Dikgale, 

# Requires Changes 🍀
You have done a great job so far!👏👏
You still need to work on a few issues before moving on to the next project, but you're almost there🪲!

## Highlights🎯
- Nice job connecting your application to the API ✅
- App works as expected ✅


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🪶
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BertrandConxy — APPROVED** [review]
Hi @Naledi-Dikgale , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💪 
- No linter errors
- project requirements are met
- project works fine
- good PR description
- correct flow is used

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**